apps/freeablo/faworld/actor.h

```diff
@@ -62,26 +91,25 @@ namespace FAWorld
                   const std::string& dieAnimPath=""
                   );
 
-            virtual void update(bool noclip, size_t ticksPassed);
+            virtual void update(bool noclip);
```

Comment:
What's the reasoning behind removing the ticks?  I think it's useful to have that in update methods to have some sort of time flow.